lib/three_scale_api/tools.rb

```diff
@@ -52,8 +52,8 @@ def self.log_factory(log_level: 'debug')
     #
     # @param [Hash] response Response from server
     def self.check_response(response)
-      if (response.is_a? Hash) && !response['error'].nil?
-        raise APIResponseError.new(response), response['error']
+      if (response.is_a? Hash) && (!response['error'] || !response['errors'])
```

Comment:
This breaks the logic. Before you had `!response['error'].nil?` and now `!response['error']`. Removing `!` would keep it.